src/trials/welcome.js

```diff
@@ -2,7 +2,7 @@ import { lang } from '../config/main'
 import { photodiodeGhostBox } from '../lib/markup/photodiode'
 import { baseStimulus } from '../lib/markup/stimuli'
 
-const pleaseBiggen = () => {
+const pleaseBegin = () => {
```

Comment:
this one isn't a typo (though perhaps not well named either) - it's asking the user to make the window larger

Reply:
Haha, I see. Fixed!